openmetadata-ui/src/main/resources/ui/src/utils/TimeUtils.ts

```diff
@@ -292,3 +292,12 @@ export const getFormattedDateFromSeconds = (
   timeStamp: number,
   format?: string
 ) => DateTime.fromSeconds(timeStamp || 0).toFormat(format || 'dd/MMM HH:mm');
+/**
+ * It takes a timestamp in milliseconds and returns a formatted date string
+ * @param {number} timeStamp - The timeStamp in milliseconds.
+ * @param {string} [format] - The format of the date you want to get default format is 'dd/MMM'.
+ */
+export const getFormattedDateFromMilliSeconds = (
+  timeStamp: number,
+  format?: string
+) => DateTime.fromMillis(timeStamp || 0).toFormat(format || 'dd/MMM');
```

Comment:
Let's format using this and use it across the app.
@Sachin-chaurasiya Create an issue to update all the date across the application
https://moment.github.io/luxon/api-docs/index.html#datetimedatetime_med

Reply:
@chirag-madlani , I did not get this part, can you suggest something?
> Let's format using this and use it across the app.

Reply:
@Sachin-chaurasiya Did you check the link above?

Reply:
Yes @chirag-madlani.

Reply:
@chirag-madlani , You meant something like this?
```js
export const getFormattedDateFromMilliSeconds = (
  timeStamp: number,
) => DateTime.fromMillis(timeStamp || 0).toLocaleString(DateTime.DATETIME_MED);
```